website/docs/core-concepts/connecting-to-data-sources/connecting-to-databases.md

```diff
@@ -5,59 +5,78 @@ description: >-
 ---
 
 # Databases
- You can connect to, and run queries to read and write data directly from the
-  Appsmith editor.
 
-## Supported Databases
+Databases are an essential part of many applications and are used to store and manage data. They provide a structured way to organize, retrieve, and manipulate data, and are designed to be efficient and reliable. You can run queries to read and write data directly from the Appsmith editor. 
 
-* [Amazon S3 (also Upcloud, Digital Ocean Spaces, Wasabi, DreamObjects)](../../reference/datasources/querying-amazon-s3.md)
-* [ArangoDB](../../reference/datasources/querying-arango-db.md)
-* [DynamoDB](../../reference/datasources/querying-dynamodb.md)
-* [ElasticSearch](../../reference/datasources/querying-elasticsearch.md)
-* [Firestore](../../reference/datasources/querying-firestore.md)
-* [Google Sheets](../../reference/datasources/querying-google-sheets.md)
-* [MongoDB](../../reference/datasources/querying-mongodb/)
-* [MS SQL](../../reference/datasources/querying-mssql.md)
-* [MySQL](../../reference/datasources/querying-mysql.md)
-* [PostgreSQL](../../reference/datasources/querying-postgres.md)
-* [Redis](../../reference/datasources/querying-redis.md)
-* [Redshift](../../reference/datasources/querying-redshift.md)
-* [Snowflake](../../reference/datasources/querying-snowflake-db.md)
-* [SMTP](../../reference/datasources/using-smtp.md)
 
-## Security
 
-Appsmith safely encrypts all your database credentials and stores them securely. Appsmith also does not store any data returned from your data sources and acts only as a proxy layer to orchestrate the API / Query calls. Since Appsmith is an open-source framework, you can [deploy it on-premise](../../getting-started/setup/), and audit it to ensure none of your data leaves your VPC.
+### Connecting to a database
 
-## Connecting to a Database
 
-:::note
-Before connecting to a data source, you may need to whitelist the IP address of the Appsmith deployment on your database instance or VPC.
+:::info important
+
+If you are a self-hosted Appsmith user, it's necessary to whitelist the IP addresses of the Appsmith deployment on your database instance or VPC. 18.223.74.85 and 3.131.104.27 are the IP addresses of the Appsmith cloud instances that need to be whitelisted.
+:::
+
+<VideoEmbed host="youtube" videoId="sJIxtXInV14" title="How to connect to a datasource" caption="How to connect to a datasource"/>
+
+* On the **Explorer tab**, click the **+** sign next to **Datasources**. 
+* Choose a Database from the [Datasources](/reference/datasources/) that Appsmith can connect to.
+* Provide the **configuration details** required to connect to your database. 
+* Click **Test** to verify that Appsmith can connect to your database using the details you provided.
+* Rename and save your datasource.
```

Comment:
Can we please add a line stressing on the importance of clicking the save button here - i.e. unless the save button is clicked the datasource configuration is not updated. Many users get confused here thinking that since the `Test` button shows that datasource is valid they can just go ahead and create a query.

Reply:
done